xarray/tests/test_backends.py

```diff
@@ -2172,6 +2179,10 @@ def test_open_zarr_use_cftime(self):
             ds_b = xr.open_zarr(store_target, consolidated=True, use_cftime=True)
             assert xr.coding.times.contains_cftime_datetimes(ds_b.time)
 
+    @requires_dask
```

Comment:
Use xfail instead, because this is a real limitation with the current implementation.